KnownIssues.md

```diff
@@ -1,63 +1,59 @@
-The following list are known issues. The issue may contain a workaround or an associated Issue number.
+## Known Issues for Oracle WebLogic Server Deploy Tooling
 
-**ISSUE**:
-   The createDomain and updateDomain tools cannot target non-JRF product resources to dynamic clusters because
-   WebLogic WLST will not assign resources associated with extension or custom template user server groups
-   to dynamic clusters. Resources associated to JRF user server groups defined in the domain typedef
-   (i.e. JRF, RestrictedJRF) will be targeted to the dynamic cluster by the WDT tools using the FMW WLST function
-   applyJRF.
+The following list are known issues. The issue may contain a workaround or an associated issue number.
 
-   If you have only non-JRF user server groups targeted to a dynamic cluster, you will see the following message:
+### Resource Assignment to Dynamic Clusters
 
-    WLSDPLY-12238: Unable to target non-JRF template server groups for domain type <your domain typedef> to dynamic cluster(s).
+**ISSUE**:
+The createDomain and updateDomain tools cannot target non-JRF product resources to dynamic clusters because WebLogic WLST will not assign resources associated with extension or custom template user server groups to dynamic clusters. Resources associated to JRF user server groups defined in the domain typedef (i.e. `JRF`, `RestrictedJRF`) will be targeted to the dynamic cluster by the WDT tools using the FMW WLST function `applyJRF`.
```

Comment:
This issue has been resolved. you can now target server groups to the dynamic cluster

Reply:
Removed this section